common/autoware_ad_api_msgs/operation_mode/msg/OperationMode.msg

```diff
@@ -0,0 +1,9 @@
+# constants for mode
```

Comment:
[Question] Are there any reasons for splitting `OperationMode` from `OperationModeState`?
It seems `OperationMode` isn't used directly, so I think it can be embedded in `OperationModeState` (at least for now).

Reply:
It is assumed that other messages refer to it like this PR https://github.com/tier4/tier4_autoware_msgs/pull/46.

Reply:
I see, thanks.